packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -217,8 +217,14 @@ export type SubmitSummaryResult =
 /** The stages of Summarize, used to describe how far progress succeeded in case of a failure at a later stage. */
 export type SummaryStage = SubmitSummaryResult["stage"] | "unknown";
 /** The data in summarizer result when submit summary stage fails. */
-export interface SubmitSummaryFailureData {
-	stage: SummaryStage;
+
+export interface IBaseSummaryFailureResult {
```

Comment:
Why do you need to split the base out from under `ISubmitSummaryFailureResult` itself?

Reply:
Ok I see, summaryOpBroadcasted doesn't need retry stuff.

Reply:
Still, since the retry stuff is optional, if it works to merge them it'd be nice, less is more

Reply:
I changed this. Please take a look now.